test/validation/validation.jl

```diff
@@ -83,4 +83,27 @@
         # Verify number of plots
         @test length(axs_edac[1].scene.plots) >= 2
     end
+
+    @trixi_testset "TGV_2D" begin
+        @trixi_test_nowarn trixi_include(@__MODULE__,
+                                         joinpath(validation_dir(),
+                                                  "taylor_green_vortex_2d",
+                                                  "validation_taylor_green_vortex_2d.jl")) [
```

Comment:
```suggestion
                                                  "validation_taylor_green_vortex_2d.jl", tspan=(0.0, 0.01)) [
```
We already test TGV in the examples. This test here is only to verify that there are no bugs in the validation files (for now there is no validation).